test/src/tests-identity.ts

```diff
@@ -4799,11 +4799,11 @@ describe('identity', function() {
             waitForCondition(hasIdentifyReturned)
             .then(() => {
 
-            const product = mParticle.eCommerce.createProduct(
+            const product: Product = mParticle.eCommerce.createProduct(
```

Comment:
I assume this and the below should all be `SDKProduct` and not `Product`
```suggestion
            const product: SDKProduct = mParticle.eCommerce.createProduct(
```

Reply:
this comment is still pending